azurerm/internal/services/compute/linux_virtual_machine_scale_set_resource.go

```diff
@@ -896,7 +896,13 @@ func resourceArmLinuxVirtualMachineScaleSetRead(d *schema.ResourceData, meta int
 		d.Set("max_bid_price", maxBidPrice)
 
 		d.Set("eviction_policy", string(profile.EvictionPolicy))
-		d.Set("priority", string(profile.Priority))
+
+		// the service just return empty when this is not assigned when provisioned
```

Comment:
fwiw this is a bug in the API that should be fixed - can we open a swagger issue for this?

Reply:
Upstream issue has opened: https://github.com/Azure/azure-rest-api-specs/issues/10971